halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/services/v1/AccountService.java

```diff
@@ -37,36 +38,42 @@
 @Component
 public class AccountService {
   @Autowired
-  ProviderService providerService;
+  LookupService lookupService;
 
   @Autowired
   ValidateService validateService;
 
-  public Account getAccount(NodeReference reference) {
-    Provider provider = providerService.getProvider(reference);
+  public List<Account> getAccounts(NodeReference reference) {
```

Comment:
I'm thinking of changing this to `getAccountNames` for now, since from an API perspective that's more useful (at least for the CLI).